books/epub.py

```diff
@@ -149,13 +150,58 @@ def get_info(self):
         return self._info
 
     def get_cover_image_path(self):
-        if self._info.cover_image is not None:
-            self._unzip_file(self._basepath + self._info.cover_image)
-            return os.path.join(self._tempdir, self._basepath,
-                                self._info.cover_image)
-        else:
+        """
+        Epubinfo returns cover_image as either an image or xhtml. If xhtml,
+        parse to find image
+        """
+        if not self._info.cover_image:
             return None
 
+        self._unzip_file(self._basepath + self._info.cover_image)
+        cover_image = os.path.join(self._tempdir,
+                                   self._basepath,
+                                   self._info.cover_image)
+
+        if self._info.cover_image.endswith(('.gif', '.jpg', '.jpeg', '.png')):
```

Comment:
Very-minor-thing: can you make the comparison case insensitve, using `self._info.cover_image.lower` or similar?